### DISCUSSION TITLE
Is there a possibility to set reminders with absolute time?

### DISCUSSION THREAD

**Discussion by acidrums4** [kuzmoyev/google-calendar-simple-api#183] [Q&A] [answered]
Being fully aware that the `Reminder` object offers a parameter `minutes_before_start`, but is there any possibility to set a reminder with an absolute time (say, 1 day before at 7:00 a.m.), sort of like what the web client offers? Not sure if this can be actually done or if there's an upstream restriction to do so.

![Screenshot_20240414_093203](https://github.com/kuzmoyev/google-calendar-simple-api/assets/513421/9a4de7c6-b32e-42bf-a802-69d96b7c8a42)

**Comment by kuzmoyev** [comment, marked as answer]
Hey @acidrums4,

Unfortunately, this format is not supported directly by [google's API](https://developers.google.com/calendar/api/v3/reference/events/insert). The web client seems to convert this to minutes before requesting the API. So your screenshot example translates to:

`PopupReminder(minutes_before_start=900)`


Having said that, I believe it can be a nice feature of GCSA. I'll move it to issues to implement it in the next version.

Thanks for the submission!

**Reply by acidrums4** [reply]
Hello there, thank you so much for answering so quickly! Looking forward for that next version. Thank you so much, I really appreciate it. Cheers

**Reply by kuzmoyev** [reply]
Hi @acidrums4 once again :)

I've added an option to set absolute time in [this commit](https://github.com/kuzmoyev/google-calendar-simple-api/pull/182/commits/38e776360ac0ca2adbc0410b1c4f00d632c89d4b). (Will release soon).

If you are interested, check the docs for it [here](https://google-calendar-simple-api--182.org.readthedocs.build/en/182/reminders.html#specific-time-reminders). Feel free to share your thoughts :)
